From c4a2195febbd5d436f8de79d8391d8da9aa60ac4 Mon Sep 17 00:00:00 2001
From: Michael Walle <michael@walle.cc>
Date: Thu, 10 Oct 2019 14:58:16 +0200
Subject: [PATCH] CMakeLists.txt: respect CMAKE_INSTALL_PREFIX and DESTDIR

Don't use absolute pathnames in the install function. This way the files
will be installed in the correct location even if CMAKE_INSTALL_PREFIX
and DESTDIR are changed.

Patch comes from upstream pull-request:
https://github.com/sysrepo/sysrepo/pull/1638

Signed-off-by: Heiko Thiery <heiko.thiery@gmail.com>
---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index e4521088..14b7cd36 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -357,7 +357,7 @@ if(WITH_SYSTEMD)
         ${PROJECT_SOURCE_DIR}/deploy/systemd/sysrepod.service
         ${PROJECT_SOURCE_DIR}/deploy/systemd/sysrepo-plugind.service
     )
-    install(DIRECTORY ${PROJECT_SOURCE_DIR}/deploy/systemd/ DESTINATION "/lib/systemd/system"
+    install(DIRECTORY ${PROJECT_SOURCE_DIR}/deploy/systemd/ DESTINATION "lib/systemd/system"
         FILES_MATCHING PATTERN "*.service")
 endif()
 
-- 
2.20.1

